tests/tests-core/src/main/java/com/gentics/mesh/cache/ActionContextBranchCacheTest.java

```diff
@@ -38,12 +40,15 @@ private ProjectBranchNameCache cache() {
 	}
 
 	@Test
-	public void testBranchCacheResetOnNewLatest() {
-		BranchResponse branch = call(() -> client().createBranch(projectName(), new BranchCreateRequest().setName("new").setLatest(false)));
-		assertNotNull(branch.getUuid());
-		assertEquals(branch.getName(), "new");
-		assertFalse(branch.getLatest());
-		String branchUuid = branch.getUuid();
+	public void testBranchCacheResetOnNewLatest() throws InterruptedException {
+		String branchUuid = UUIDUtil.randomUUID();
+		waitForJobs(() -> {
+			BranchResponse branch = call(() -> client().createBranch(projectName(), branchUuid, new BranchCreateRequest().setName("new").setLatest(false)));
```

Comment:
Flawed test - one has to wait for the branch migration. By now we were lucky to pass the race with the migration job, but Hib 6.x is faster...